metaflow/metaflow_environment.py

```diff
@@ -55,6 +55,13 @@ def add_to_package(self):
         """
         return []
 
+    # TODO: this must be unified as '1 linter config' for pylint, ruff
+    def ruff_config(self):
```

Comment:
these shall be united/called basis the choice of the linter, seeking advice here @tuulos